UndertaleModTool/Controls/UndertaleRoomRenderer.xaml

```diff
@@ -235,7 +235,11 @@
                     <Rectangle Width="{Binding Width, Mode=OneTime}" Height="{Binding Height, Mode=OneTime}"
                                Opacity="{Binding Color, Mode=OneTime, Converter={StaticResource ColorToOpacityConverter}}">
                         <Rectangle.RenderTransform>
-                            <TranslateTransform x:Name="transform2_0" X="{Binding X, Mode=OneTime}" Y="{Binding Y, Mode=OneTime}"/>
+                            <TransformGroup>
+                                <TranslateTransform x:Name="transform2_0" X="{Binding X, Mode=OneWay}" Y="{Binding Y, Mode=OneWay}"/>
```

Comment:
Is there a reason this was changed from `OneTime` to `OneWay`? I'm guessing that was done that way for a performance reason, just not totally sure.